Jenkinsfile

```diff
@@ -94,7 +94,33 @@ pipeline {
                     '''
                 }
             }
-        }  
+        }
+
+        stage('Run Turbine Unit Test Suite') {
+            options {
+                timeout(time: 30, unit: 'MINUTES') 
+            }
+                steps {
+                    withEnv(["PATH=$PATH:~/.local/bin;NOBUILD=true"]) {
+                        withDockerRegistry([url: 'https://index.docker.io/v1/', credentialsId: 'docker.com-bot']) {
+                            sh '''#!/usr/bin/env bash
+
```

Comment:
Looks like you need to export the node path, it is complaining not finding npm https://ci.eclipse.org/codewind/job/Codewind/job/codewind/job/PR-554/4/execution/node/31/log/
```
export PATH=$PATH:/home/jenkins/.jenkins/tools/jenkins.plugins.nodejs.tools.NodeJSInstallation/node_js/bin/
```